liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/PluginConfigSupport.java

```diff
@@ -242,49 +243,47 @@ protected String stripVersionFromName(String name, String version) {
 
     protected boolean isAppConfiguredInSourceServerXml(String fileName) {
 
-        boolean bConfigured = false;
+    	boolean bConfigured = false; 
 
-        // get server.xml source
-        File serverXML = new File(serverDirectory, "server.xml");
-
-        if (serverXML != null && serverXML.exists()) {
-            try {
-                ServerConfigDocument scd = ServerConfigDocument.getInstance(log, serverXML, configDirectory,
-                        bootstrapPropertiesFile, bootstrapProperties, serverEnv);
+    	Set<String> locations = getAppConfigLocationsFromSourceServerXml();
 
-                if (scd != null && scd.getLocations().contains(fileName)) {
-                    log.debug("Application configuration is found in server.xml : " + fileName);
-                    bConfigured = true;
-                }
-            } catch (Exception e) {
-                log.warn(e.getLocalizedMessage());
-                log.debug(e);
-            }
+        if (locations.contains(fileName)) {
+            log.debug("Application configuration is found in server.xml : " + fileName);
+            bConfigured = true;
         }
+        
         return bConfigured;
     }
 
     protected boolean isAnyAppConfiguredInSourceServerXml() {
 
-        boolean bConfigured = false;
+    	boolean bConfigured = false; 
+
+    	Set<String> locations = getAppConfigLocationsFromSourceServerXml();
+        if (locations.size() > 0) {
+            log.debug("Application configuration is found in server.xml.");
+            bConfigured = true;
```

Comment:
Here I would just say `return true`. Then fall through `return false`, instead of creating a variable.